docs/docs/support/troubleshooting.mdx

```diff
@@ -73,3 +73,40 @@ The WebFinger requirement and setup is a step a user has to take outside of thei
 On their custom domain, e.g example.com, users need to host a WebFinger endpoint at https://example.com/.well-known/webfinger. When queried, this endpoint returns a JSON response detailing the issuer. Users would need to host the endpoint with the link to the ZITADEL issuer. Tailscale only looks up this endpoint once when a user signs up, and will only look up this endpoint again if the user needs to make a configuration change to their identity provider.
 
 The requirements and a set up guide is detailed in the [Tailscale documentation](https://tailscale.com/kb/1240/sso-custom-oidc/).
+
+## Login not possible. The organization of the user must be granted to the project
+
+![Organization must be granted Error](/img/support/login_not_possible_error.png)
+
+ZITADEL does is not only capable of handling authentication but also authorization.
+This error message tells you, that a permission is missing for the organization of the user.
```

Comment:
```suggestion
The error message "Login not possible. The organization of the user must be granted to the project." tells you, that a permission is missing for the organization of the user. 
```

Reply:
Which user? I assume the user trying to login to the project?

Reply:
Shouldn't we say "project grant" is missing instead of "permission"?

Reply:
changed